src/containers/CheckoutPage.jsx

```diff
@@ -426,7 +426,7 @@ class CheckoutPage extends React.Component {
               </div>
               {deletable && (
                 <div className="payment-btn-cell">
-                  <button className="btn checkout-delete-payment-btn" type="button" onClick={() => { this.handleDelete(payment.self.uri); }}>
+                  <button className="vestri-btn small" type="button" onClick={() => { this.handleDelete(payment.self.uri); }}>
```

Comment:
The independent class names (example: checkout-delete-payment-btn) are meant more for readability and for automation. I'd prefer to keep some or add IDs where applicable rather than having generic elements in some places.